infrastructure/main.tf

```diff
@@ -0,0 +1,21 @@
+provider "azurerm" {
+  features {}
+}
+
+module "resource_group" {
+  source = "./resource_group"
+
+  resource_group_name = var.resource_group_name
+  location = var.location
+}
+
+module "storage_account" {
+  source = "./storage_account"
+
+  depends_on = [module.resource_group]
+
+  resource_group_name = var.resource_group_name
+  location = var.location
+  name = "mindgptdvc"
+  container_name = "mindgptdvccontainer"
```

Comment:
Note that these names must be unique across Azure (regions?) hence another person will not be able to provision this whilst it exists on another account. We should think about either using some random hash on the end or exporting this as an environment variable.

Reply:
I think this is solved with documentation if we're adding it re setting up new storage

Reply:
I've made these variables and pointed the user to variables.tf duringg DVC set up in docs.